core/auth/src/main/java/software/amazon/awssdk/auth/signer/internal/Aws4SignerRequestParams.java

```diff
@@ -141,4 +129,25 @@ public String getFormattedSigningDate() {
     public String getSigningAlgorithm() {
         return SignerConstant.AWS4_SIGNING_ALGORITHM;
     }
+
+    private Clock resolveSigningClock(Aws4SignerParams signerParams) {
+        if (signerParams.signingClockOverride().isPresent()) {
+            return signerParams.signingClockOverride().get();
+        }
+        Clock baseClock = Clock.systemUTC();
```

Comment:
For clarification, do we always use time in UTC?

Reply:
Yeah, the previous code used `System.currentTimeMillis()` to calculate the request date if no override was given, which is in UTC.